geode-core/src/distributedTest/java/org/apache/geode/cache/ConnectionPoolDUnitTest.java

```diff
@@ -1759,199 +1224,162 @@ public void run2() throws CacheException {
         }
       }
     });
-    vm2.invoke("recreate", new CacheSerializableRunnable() {
-      @Override
-      public void run2() throws CacheException {
-        Region<Object, Object> region = getRootRegion().getSubregion(name);
-        for (int i = 0; i < 10; i++) {
-          Object key = i;
-          region.create(key, "create" + i);
-        }
+    vm2.invoke("recreate", () -> {
+      Region<Object, Object> region = getRootRegion().getSubregion(name);
+      for (int i = 0; i < 10; i++) {
+        Object key = i;
+        region.create(key, "create" + i);
       }
     });
-    Wait.pause(5 * 1000);
 
-    vm1.invoke("Verify creates", new CacheSerializableRunnable() {
-      @Override
-      public void run2() throws CacheException {
-        Region<Object, Object> region = getRootRegion().getSubregion(name);
-        CertifiableTestCacheListener ctl =
-            (CertifiableTestCacheListener) region.getAttributes().getCacheListener();
-        List l = ctl.getEventHistory();
-        logger
-            .info("history (should be empty): " + l);
-        assertEquals(0, l.size());
-        // now see if we can get it from the server
-        for (int i = 0; i < 10; i++) {
-          Object key = i;
-          assertEquals("create" + i, region.get(key, "loadCB" + i));
-        }
-        l = ctl.getEventHistory();
-        assertEquals(10, l.size());
-        for (int i = 0; i < 10; i++) {
-          Object key = i;
-          EntryEvent ee = (EntryEvent) l.get(i);
-          logger.info("processing " + ee);
-          assertEquals(key, ee.getKey());
-          assertNull(ee.getOldValue());
-          assertEquals("create" + i, ee.getNewValue());
-          assertEquals(Operation.LOCAL_LOAD_CREATE, ee.getOperation());
-          assertEquals("loadCB" + i, ee.getCallbackArgument());
-          assertFalse(ee.isOriginRemote());
-        }
+    vm1.invoke("Verify creates", () -> {
+      Region<Object, Object> region = getRootRegion().getSubregion(name);
+      CertifiableTestCacheListener<Object, Object> ctl =
+          (CertifiableTestCacheListener<Object, Object>) region.getAttributes()
+              .getCacheListeners()[0];
+      List<CacheEvent<Object, Object>> list = ctl.getEventHistory();
+      logger
+          .info("history (should be empty): " + list);
+      assertThat(0).isEqualTo(list.size());
+      // now see if we can get it from the server
+      for (int i = 0; i < 10; i++) {
+        Object key = i;
+        assertThat("create" + i).isEqualTo(region.get(key, "loadCB" + i));
+      }
+      list = ctl.getEventHistory();
+      assertThat(10).isEqualTo(list.size());
+      for (int i = 0; i < 10; i++) {
+        Object key = i;
+        EntryEvent ee = (EntryEvent) list.get(i);
+        logger.info("processing " + ee);
+        assertThat(key).isEqualTo(ee.getKey());
+        assertThat(ee.getOldValue()).isNull();
+        assertThat("create" + i).isEqualTo(ee.getNewValue());
```

Comment:
I would flip this one around too:
```
assertThat(ee.getNewValue()).isEqualTo("create" + i);
```
And flip around the next couple assertions as well.